notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue

```diff
@@ -239,7 +239,7 @@
       :default-path="defaultPath"
       @open-treeview="$refs.notesBreadcrumb.open(note, 'movePage')"
       @export-pdf="createPDF(note)"
-      @open-history="$refs.noteVersionsHistoryDrawer.open(noteVersions,isManager)"
+      @open-history="openNoteVersionsHistoryDrawer(noteVersions, isManager)"
```

Comment:
function openNoteVersionsHistoryDrawer does not take parameters, could you clean the invocation ?